WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -143,7 +147,9 @@ public Simulation ToServiceModel(Simulation existingSimulation, string id = "")
         }
 
         // Map service model to API model
-        public static SimulationApiModel FromServiceModel(Simulation value)
+        public static SimulationApiModel FromServiceModel(
```

Comment:
this signature is inconsistent with the usual `FromServiceModel` functions.  What about `SimulationStatisticsModel` being an optional field inside the `Simulation` class instead? See also my comment in controller